docs/specs/om/open_metrics_spec_1_1.md

```diff
@@ -118,6 +118,8 @@ A MetricFamily MAY have zero or more Metrics. A MetricFamily MUST have a name, H
 
 MetricFamily names are a string and MUST be unique within a MetricSet. Names SHOULD be in snake_case. Metric names MUST follow the restrictions in the ABNF section.
 
+Ingestors MAY accept duplicate MetricFamily names, provided that either of HELP, TYPE, or UNIT is unique within the duplicated MetricFamily names. Notably Exposers MAY duplicate MetricFamilies when aggregating external metrics that have the same MetricFamily name, but come from different sources and have either different HELP, or TYPE, or UNIT. Be aware that exposing such metrics directly to end-users reduces usability due to confusion about which Metric belongs to which MetricFamily.
```

Comment:
IMO we can still require metric family names to be unique within the text exposition format while being more tolerant of other naming. It properly places the burden of ensuring unique metric family names on the exposers that choose to deviate from naming recommendations (e.g. OTel SDKs), and doesn't impose an additional burden on Ingestors.

Reply:
Not allowing federation/push-gateway isn't a regression, it has always been the case. I'll remove these additions from the PR.